### PR TITLE
Conn 2224 - move RLS-table

### PR DESCRIPTION
**Pull request by trantang**
paul done the work
updated-validationSuite to passed

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/persistence/HibernateUtilFactory.java [hunk 1]
```diff
@@ -175,9 +176,20 @@ private ClassPathSingleton() {
...
-                    gov.hhs.fha.nhinc.direct.messagemonitoring.persistence.HibernateUtil.class);
+                gov.hhs.fha.nhinc.direct.messagemonitoring.persistence.HibernateUtil.class);
         }
         return msgMonitorHibernateUtil;
     }
 
+    public static gov.hhs.fha.nhinc.patientdiscovery.persistence.HibernateUtil getPatientDiscoveryHibernateUtil() {
+        ClassPathXmlApplicationContext context = ClassPathSingleton.CONTEXT;
+
+        LOG.debug("Memory address getPatientDiscoveryHibernateUtil {}", context.getId());
```
Comment (plobre) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/persistence/HibernateUtilFactory.java]: Do we need this?

File: Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/patientlocationquery/adapter/dao/PatientLocationQueryDaoIntTest.java [hunk 2]
```diff
@@ -35,11 +35,10 @@
 import gov.hhs.fha.nhinc.patientlocationquery.dao.RecordLocationServiceDAO;
 import gov.hhs.fha.nhinc.test.DAOIntegrationTest;
 import java.util.List;
-import org.junit.Ignore;
 import org.junit.Test;
 
 //UnIgnore to run DAO Integration test which will use a MySQL DB connection.
-@Ignore
+//@Ignore
```
Comment (plobre) [inline Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/patientlocationquery/adapter/dao/PatientLocationQueryDaoIntTest.java]: This needs to be un-commented out

**Review by plobre — CHANGES_REQUESTED** [review]

File: Product/DBScripts/nhincdb/nhincdb.sql [hunk 3]
```diff
@@ -474,6 +466,14 @@ CREATE TABLE IF NOT EXISTS patientdb.phonenumber (
         ON UPDATE NO ACTION
 ) COMMENT = 'Phone Numbers';
 
+CREATE TABLE IF NOT EXISTS patientdb.recordlocatorservice (
```
Comment (TabassumJafri) [inline Product/DBScripts/nhincdb/nhincdb.sql:469]: Update this in oracle scripts as well

**Review by TabassumJafri — CHANGES_REQUESTED** [review]

File: Product/DBScripts/nhincdb/nhincdb.sql (hunk 3 above)
Comment (trantang) [reply Product/DBScripts/nhincdb/nhincdb.sql:469]: Oracle-script had no affect on this change

**Review by TabassumJafri — APPROVED** [review]

**Review by plobre — APPROVED** [review]

**Pull request merged**